tests/size_layer_diff_expected.json

```diff
@@ -0,0 +1,24 @@
+[
+  {
+    "Image1": "gcr.io/gcp-runtimes/diff-layer-base",
+    "Image2": "gcr.io/gcp-runtimes/diff-layer-modified",
+    "DiffType": "SizeLayer",
+    "Diff": [
+      {
+        "Name": "0",
+        "Size1": 6,
+        "Size2": 6
```

Comment:
hmm. any idea why this is showing up when the two sizes are the same?

Reply:
The way I've implemented this is technically not a "diff." In both cases, image and layer size diff, it just shows a comparison between the two even if they match. Now that you've pointed it out I've realised that the diff output probably needs a rethink. It probably should only display layers where the size differs between the two images. If there are no differences and it's an exact match I'm not sure what the expected output should be. Let me know if you have some ideas.

Reply:
I think for this case, it would be fine to just not display the result at all if the two sizes are identical. Since we're looking for diffs, this is the exact opposite of a diff, so we can just omit it from the results. Then the output will just have a list of the layers that have actual size differences. Does that make sense?